src/main/java/org/osiam/resources/controller/GroupController.java

```diff
@@ -117,4 +131,13 @@ public void delete(@PathVariable final String id) {
         headers.setLocation(location);
         return new ResponseEntity<>(group, headers, status);
     }
+
+    private Set<String> extractAttributes(String attributesParameter) {
+        List<String> strings = Lists.newArrayList("schemas");
+        if (!Strings.isNullOrEmpty(attributesParameter)) {
+            strings.addAll(Arrays.asList(attributesParameter.split("[,|\\.]")));
+        }
+        strings.forEach(e -> strings.set(strings.indexOf(e), e.trim()));
```

Comment:
Please move this into the then block. It makes no sense to me, to trim the default attribute (`"schemas"`), if no attributes have been submitted.